life/updated/03-testimonies.adoc

```diff
@@ -76,61 +76,61 @@ John Thornton
 
 It is in my mind to write something as a testimony
 concerning my dear and well-beloved friend,
-Elizabeth Stirredge, deceased.
+Elizabeth Stirredge, who is now deceased.
```

Comment:
I think you can leave this saying **Elizabeth Stirredge, deceased.**  It sounds weird at first, but ALL of the testimonies in Friends Library say it this way.